tests/app/utilities/test_decimal_places.py

```diff
@@ -0,0 +1,129 @@
+import unicodedata
+from decimal import Decimal
+
+import pytest
+
+from app.utilities.decimal_places import (
+    custom_format_currency,
+    custom_format_decimal,
+    custom_format_unit,
+)
+
+
+@pytest.mark.parametrize(
+    "value, locale_string, expected_result",
+    (
+        (123, "en_GB", "123"),
+        (123, "es_ES", "123"),
+        (Decimal("123.4"), "en_GB", "123.4"),
+        (Decimal("123.4"), "es_US", "123.4"),
+        (Decimal("123.40"), "en_GB", "123.40"),
+        (Decimal("123.400"), "en_GB", "123.400"),
+        (Decimal("123.4000"), "en_GB", "123.4000"),
+        (Decimal("123.4000"), "pl_PL", "123,4000"),
+        (Decimal("123.40000"), "en_GB", "123.40000"),
+        (Decimal("123.40000"), "ja_JP", "123.40000"),
+        (Decimal("123434.7678"), "en_GB", "123,434.7678"),
+        (Decimal("123434.7678"), "hu_HU", "123 434,7678"),
+        (Decimal("123.45678"), "en_GB", "123.45678"),
+        (Decimal("2344.6533"), "en_GB", "2,344.6533"),
+        (1000, "en_GB", "1,000"),
+        (1000, "en_US", "1,000"),
+        (10000, "en_GB", "10,000"),
+        (10000, "es_ES", "10.000"),
+        (100000000, "en_GB", "100,000,000"),
+        (0, "en_GB", "0"),
+        (0, "de_DE", "0"),
+        (Decimal("0.00"), "en_GB", "0.00"),
+        (Decimal("0.000"), "en_GB", "0.000"),
+        (Decimal("0.000"), "es_ES", "0,000"),
+        (Decimal("0.00000"), "en_GB", "0.00000"),
+        (Decimal("0.00000"), "es_ES", "0,00000"),
+    ),
+)
```

Comment:
Why is this different to `TEST_FORMAT_NUMBER_PARAMS`, they both use `custom_format_decimal`. The setting of locale can be done by the test, so you could use this for both?.

Reply:
Updated 👍